Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM golang:1.10.1-alpine3.7
 
-RUN apk add --no-cache build-base git linux-headers python python-dev py-pip py-setuptools
+ARG GOVMOMI_CHECKOUT=dee49fa3694c5aff05e4b340b0686772f65c1fe1
 
 ADD requirements.txt /root/requirements.txt
-RUN pip install -r /root/requirements.txt
-ADD flask_control.py /root/flask_control.py
 
-RUN go get -d github.com/vmware/govmomi && \
-    cd ${GOPATH}/src/github.com/vmware/govmomi && \
-    git checkout dee49fa3694c5aff05e4b340b0686772f65c1fe1 && \
-    go install github.com/vmware/govmomi/govc github.com/vmware/govmomi/vcsim
+RUN apk add --no-cache build-base git linux-headers python python-dev py-pip py-setuptools; \
```

Comment:
Use `&&` instead of `;` between commands. That will make sure the build stops after the first command fails instead of continuing on, possibly building a non-functional container.

Reply:
thanks @mattclay switched back to ``&&``